syft/pkg/language.go

```diff
@@ -13,6 +13,7 @@ const (
 	// the full set of supported programming languages
 	UnknownLanguage Language = ""
 	Java            Language = "java"
+	JvmBytecode     Language = "jvm-bytecode"
```

Comment:
Is this decidedly a separate language? Would we typically be getting Java dependencies in the embedded information? I have a suspicion that this could result in missed java vulnerabilities if it's not the `java` language.

Reply:
We would typically see Java dependencies in the SBOM, so we can change the language to `java` in order to avoid missing relevant vulnerabilities.